src/components/ebay-area-chart/area-chart.stories.js

```diff
@@ -0,0 +1,88 @@
+import { addRenderBodies } from '../../../.storybook/utils';
+import Readme from './README.md';
+import Component from './index.marko';
+
+function getPositiveTrend(days) {
```

Comment:
This is good, but when users are looking through our docs, we want them to easily find how to add data. 
Maybe we should have this as a separate file? Or we can add a template with all the data hardcoded. (See button for examples on that https://github.com/eBay/ebayui-core/blob/master/src/components/ebay-button/button.stories.js#L5-L6)